zanata-war/src/main/webapp/language/language.xhtml

```diff
@@ -140,6 +177,17 @@
         </div>
       </div>
     </div>
+    <s:fragment rendered="#{identity.loggedIn}">
+      <ui:include src="../WEB-INF/layout/language/contact_coordinator_modal.xhtml">
+        <ui:param name="id" value="#{languageAction.language}" />
+        <ui:param name="displayName" value="#{languageAction.locale.retrieveDisplayName()}" />
+      </ui:include>
+      <s:fragment rendered="#{not languageAction.isUserInTeam()}">
+        <ui:include src="../WEB-INF/layout/language/request_join_modal.xhtml">
```

Comment:
This should not need to use `../WEB-INF`, should work with just `/WEB-INF` and will be more robust.